app/feature.go

```diff
@@ -0,0 +1,133 @@
+package app
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/urfave/cli/v2"
+)
+
+var (
+	ExportFeatureFlag   = "enable-export"
+	featureflagFileName = "feature.json"
+)
+
+type FeatureFlag struct {
+	Name  string `json:"Name"`
+	Value bool   `json:"Value"`
+}
+
+func getFeatureFlagJSON() ([]FeatureFlag, error) {
+	featureFlagConfigPath := getFeatureFlagConfigPath()
+	return getFeatureFlagConfig(featureFlagConfigPath)
+}
+
+func getFeatureFlagConfigPath() string {
+	return filepath.Join(ConfigDirFlag.Value, featureflagFileName)
+}
+
+func getFeatureFlagConfig(featureFlagConfigPath string) ([]FeatureFlag, error) {
+	// create config file if not exist
+	if _, err := os.Stat(featureFlagConfigPath); err != nil {
+		if err := ioutil.WriteFile(featureFlagConfigPath, []byte("[]"), 0644); err != nil {
+			return nil, err
+		}
+	}
+	content, err := ioutil.ReadFile(featureFlagConfigPath)
+	if err != nil {
+		return nil, err
+	}
+
+	var jsonData []FeatureFlag
```

Comment:
[Nit] rename variable to featureFlags. you can rename content to 'json data' if you'd like